static/app/views/performance/traceDetails/content.tsx

```diff
@@ -94,14 +99,65 @@ class TraceDetailsContent extends React.Component<Props, State> {
   }
 
   renderTraceNotFound() {
-    const {meta} = this.props;
+    const {meta, traceEventView, traceSlug, organization, location} = this.props;
 
     const transactions = meta?.transactions ?? 0;
     const errors = meta?.errors ?? 0;
 
     if (transactions === 0 && errors > 0) {
+      const eventView = {...traceEventView};
+      eventView.query = `trace:${traceSlug} !event.type:transaction `;
+
       return (
-        <LoadingError message={t('The trace you are looking contains only errors.')} />
+        <React.Fragment>
+          <Alert type="error" icon={<IconInfo size="md" />}>
+            <ErrorLabel>
+              {t('The trace cannot be shown when all events are errors.')}
+            </ErrorLabel>
+            <DiscoverQuery
+              eventView={traceEventView}
+              orgSlug={organization.slug}
+              location={location}
+              referrer="api.trace-view.errors"
+            >
+              {({isLoading, tableData, error}) => {
+                if (isLoading) {
+                  return (
+                    <Layout.Main fullWidth>
+                      <LoadingIndicator />
+                    </Layout.Main>
+                  );
+                }
+
+                if (error) {
+                  return (
+                    <ErrorLabel>
+                      {t(`An error occurred when trying to fetch error events: ${error}`)}
+                    </ErrorLabel>
+                  );
+                }
+
+                return (
+                  <List symbol="bullet" data-test-id="trace-view-errors-list">
+                    {tableData.data.map(data => (
+                      <ListItem key={data.id}>
+                        {tct(`[link]: ${data.title}`, {
+                          link: (
+                            <Link
+                              to={`/organizations/${organization.slug}/discover/${data.project}:${data.id}`}
```

Comment:
It might make more sense to link to the error event within issues rather than discover.